### PR TITLE
Update Wikipedia related content

### PR DESCRIPTION
**Pull request by egekorkan**
- update td wikipedia from my own writing -> Needs review
- update wot wikipedia from wikipedia.org

### CODE REVIEW

File: wikipedia/thing_description-english.md [hunk 1]
```diff
@@ -10,7 +12,119 @@ Thing Description is a digital document that describes the metadata and interfac
...
+        "overheating":{
+            "data": {"type": "string"},
+            "forms": [{
+                "href": "mqtt://mylamp.example.com/oh",
+                "mqv:controlPacketValue": "SUBSCRIBE"
+            }]
+        }
+    }
+}
+</syntaxhighlight>Differently from the last TD, here the forms include MQTT protocol as specified by the WoT Binding Templates<ref name=":4">{{Cite web|url=https://www.w3.org/TR/wot-binding-templates/Overview.html|title=Web of Things (WoT) Protocol Binding Templates|website=www.w3.org|access-date=2019-11-17}}</ref>. More specifically, lines 17, 25 and 34 describe what message types should be used to use the affordances. For example, instead of HTTP GET and longpoll subprotocol to observe the overheating event, a client can subscribe to this event using the MQTT protocol. <br />
```
Comment (sebastiankb) [inline wikipedia/thing_description-english.md]: we should mention, that the MQTT-based lamp service is both, a MQTT publisher and a MQTT subscriber.

**Comment by egekorkan** [comment]
I have incorporated your comments and I am attaching a rendered screenshot as well.
image

There are some parts that will be automatically changed in the last version, such as the table of contents.

**Comment by egekorkan** [comment]
Could @sebastiankb and @takuki review it so that it could be discussed in the TD call of 22.11.2019 ?

File: wikipedia/thing_description-english.md [hunk 2]
```diff
@@ -10,7 +12,120 @@ Thing Description is a digital document that describes the metadata and interfac
...
+| url = https://www.w3.org/TR/2019/CR-wot-thing-description-20191106/
 | free = Yes
-| genre=[[W3C recommendation]]
+| genre = [[W3C recommendation]]
 }}
+
+
+== Thing Description Examples ==
+
+Below is an example TD serialized in JSON-LD format, which has one property, one action and one event. The IoT device represented by this TD uses the [[HTTP]] protocol but a TD can represent any protocol with a [[URI scheme]], as shown in the further examples below.<syntaxhighlight lang="json-ld" line="1">
```
Comment (danielpeintner) [inline wikipedia/thing_description-english.md]: singular: there is just one example -> "further example below."

File: wikipedia/thing_description-english.md [hunk 3]
```diff
@@ -10,7 +12,120 @@ Thing Description is a digital document that describes the metadata and interfac
...
+
+From this TD example, a client know that there exists one Property affordance with the title ''status'' (lines 10-16). In addition, information is provided in lines 13-14 that this Property is readable with an HTTP GET request to the URI <code><nowiki>https://mylamp.example.com/status</nowiki></code>, and will return a string-based status value. In a similar manner, an Action affordance is specified to toggle the switch status using the POST method on the <code><nowiki>https://mylamp.example.com/toggle</nowiki></code> resource. The Event affordance enables a mechanism for asynchronous messages to be sent by a Thing. Here, a subscription to be notified upon a possible overheating event of the lamp can be obtained by using HTTP with its long polling subprotocol on <code><nowiki>https://mylamp.example.com/oh</nowiki></code>. The use of the GET or POST method is stated explicitly but can be omitted using the default assumptions stated in the TD specification. It can be seen that the HTTP methods are defined using the <code>"htv:methodName"</code> vocabulary terms. This vocabulary terms for HTTP are included in the TD vocabulary that is found in the <code>"@context"</code> value.
```
Comment (danielpeintner) [inline wikipedia/thing_description-english.md]: "a client know" --> "a client knows"

**Comment by egekorkan** [comment]
I have incorporated your feedback @danielpeintner 
Also realized that my wikipedia page is simply available here: https://en.wikipedia.org/wiki/User:Egekorkan/sandbox

**Comment by takuki** [comment]
In Section 10 IANA Considerations of TD spec, the file name extension there is ".jsontd" whereas the extensions used in the proposed Wikipedia page is ".td" and ".json".

We need to make them consistent.

**Comment by egekorkan** [comment]
> In Section 10 IANA Considerations of TD spec, the file name extension there is ".jsontd" whereas the extensions used in the proposed Wikipedia page is ".td" and ".json".
> 
> We need to make them consistent.

Fixed with the new commit

**Comment by egekorkan** [comment]
Now submitted to review at wikipedia

**Pull request merged**